### PR TITLE
Fix double publicProof JSON

### PR DESCRIPTION
**Pull request by rabee333**

### CODE REVIEW

File: config/dev.sample.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-   "CONTRACT_ADDRESS": "0x5a76b2e98ae411e4791b9dd8f234947ade05ecf0",
```
Comment (amiturgman) [inline config/dev.sample.json:2]: this change doesn't matter just that it is not necessary. this is just a sample file. you probably updated this by mistake.

File: services/proof.js [hunk 2]
```diff
@@ -87,18 +87,20 @@ async function storeProof(opts) {
   var proofToEncryptStr = JSON.stringify(opts.proofToEncrypt);
   var hash = sha256(proofToEncryptStr);
 
-  var publicProof = JSON.stringify({
+  var publicProof = opts.publicProof;
+  publicProof.encryptedProofHash = hash.toUpperCase();
+  /*var publicProof = JSON.stringify({
```
Comment (amiturgman) [inline services/proof.js]: remove commented code

**Review by amiturgman — CHANGES_REQUESTED** [review]

File: config/dev.sample.json (hunk 1 above)
Comment (rabee333) [reply config/dev.sample.json:2]: Submitted by mistake. I'll revert it

File: services/proof.js (hunk 2 above)
Comment (rabee333) [reply services/proof.js]: done

**Pull request merged**